lib/std/fs/get_app_data_dir.zig

```diff
@@ -57,6 +51,101 @@ pub fn getAppDataDir(allocator: mem.Allocator, appname: []const u8) GetAppDataDi
     }
 }
 
+/// Parses home directory from /etc/passwd.
+fn getHomeDirFromPasswd() ![]u8 {
+    const ReaderState = enum {
+        Start,
```

Comment:
nitpick: these enum fields should be `snake_case`: https://ziglang.org/documentation/master/#Names